test/runtests.jl

```diff
@@ -35,6 +36,12 @@ end
         t_stop = 6 * 3600,
     )
     nothing
+
+    # Test running from parsed namelist
+    namelist_path = "Output.Bomex.01/namelist_Bomex.in"
+    read_namelist = JSON.parsefile(namelist_path)
+    read_namelist["meta"]["uuid"] = "02"
+    main(read_namelist)
```

Comment:
```suggestion
    # Perform single step
    read_namelist["time_stepping"]["t_max"] = read_namelist["time_stepping"]["dt_max"]
    sim = Simulation1d(read_namelist)
    initialize(sim)
    (prob, alg, kwargs) = solve_args(sim)
    integrator = ODE.init(prob, alg; kwargs...)
    ODE.step!(integrator) # perform single step
```
Since this is done in serial, maybe we can just do a single timestep instead. We might need to add `import OrdinaryDiffEq; const ODE = OrdinaryDiffEq` above, too.

Reply:
I think we can directly call main after modifying get the `t_max`, that's what we do in CEDMF for creating the references. I'll try.